desktop/src/Desktop/Setup.hs

```diff
@@ -519,20 +518,20 @@ setPassword dSeed = form "" $ do
         )
 
   p1elem <- uiPassword $ "Enter password (" <> tshow minPasswordLength <> " character min.)"
-  p2elem <- uiPassword "Confirm password" 
+  p2elem <- uiPassword "Confirm password"
 
-  let p1 = current $ value p1elem
-      p2 = current $ value p2elem
+  p1Dirty <- holdUniqDyn =<< holdDyn False (True <$ _inputElement_input p1elem)
+  p2Dirty <- holdUniqDyn =<< holdDyn False (True <$ _inputElement_input p2elem)
 
-      inputsNotEmpty = not <$> liftA2 (||) (T.null <$> p1) (T.null <$> p2)
+  let inputsDirty = current $ liftA2 (&&) p1Dirty p2Dirty
 
-  eCheckPassword <- fmap (gate inputsNotEmpty) $ delay 0.4 $ leftmost
+  eCheckPassword <- fmap (gate inputsDirty) $ delay 0.2 $ leftmost
```

Comment:
If I'm reading this right, errors won't be shown until both fields have received input - is that intended?